Public/Sdk/Public/Managed/Testing/XUnit/xunit.dsc

```diff
@@ -74,7 +74,8 @@ export function runConsoleTest(args: TestRunArguments): Result {
 
     const qualifierRelative = r`${qualifier.configuration}/${qualifier.targetFramework}/${qualifier.targetRuntime}`;
     const parallelRelative = args.parallelBucketIndex !== undefined ? `${args.parallelBucketIndex}` : `0`;
-    const xunitLogDir = d`${Context.getMount("LogsDirectory").path}/XUnit/${Context.getLastActiveUseModuleName()}/${Context.getLastActiveUseName()}/${qualifierRelative}/${parallelRelative}`;
+    const privilege = args.privilegeLevel !== undefined ? `${args.privilegeLevel}` : `standard`;

```

Comment:
>`${args.privilegeLevel} [](start = 58, length = 23)

unncessary string formatting here..

Reply:
cleaner:
`args.privilegeLevel || "standard"`

---
In reply to: [296362769](https://github.com/microsoft/BuildXL/pull/492#discussion_r296362769) [](ancestors = 296362769)

Reply:
I thought it would result in type check error.

---
In reply to: [296362858](https://github.com/microsoft/BuildXL/pull/492#discussion_r296362858) [](ancestors = 296362858,296362769)